backend/src/messages/messages.service.spec.ts

```diff
@@ -41,9 +45,117 @@ describe('MessagesService', () => {
     expect(service).toBeDefined();
   });
 
-  it('create message in channel that does not exist', () => {
-    expect(
-      service.create({ channelId: 1, senderId: 1, content: 'Hi' }),
-    ).rejects.toThrow(EntityDoesNotExistError);
+  describe('when creating a message in a channel that does not exist', () => {
+    it('should throw', () => {
+      expect(
+        service.create({ channelId: 1, senderId: 1, content: 'Hi' }),
```

Comment:
What happens here is that by default, the `Channel` retrieved in `messages.service.ts` when doing `channelsRespository.findOne` has an undefined `id`. Hence, you are not really doing a test 'when creating a message in a channel that does not exist'.

Reply:
I could push a commit where `MockRepository(new stuff())` assigns an incremented `id` to each `stuff`.